integration/test/Test/Conversation.hs

```diff
@@ -106,20 +108,13 @@ testFederationStatus :: HasCallStack => App ()
 testFederationStatus = do
   uid <- randomUser OwnDomain def {Internal.team = True}
   federatingRemoteDomain <- asString OtherDomain
-  let unknownDomain = "foobar.com"
-  let invalidDomain = "c.example.com" -- has no srv record
+  let invalidDomain = "c.example.com" -- Does not have any srv records
   bindResponse
     (API.getFederationStatus uid [])
     $ \resp -> do
       resp.status `shouldMatchInt` 200
       resp.json %. "status" `shouldMatch` "fully-connected"
 
-  bindResponse
-    (API.getFederationStatus uid [unknownDomain])
-    $ \resp -> do
-      resp.status `shouldMatchInt` 400
-      resp.json %. "label" `shouldMatch` "discovery-failure"
-
```

Comment:
Note to reviewer: This test cannot be done in K8s because it relies on the DNS server to refuse to answer queries for a domain. This cannot happen in K8s because the DNS server there assumes that it has access to information of all domains.